incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -533,6 +533,8 @@ We are considering replacing this with the standard
 
 #### hnc.x-k8s.io/excluded-namespace (label on namespaces)
 
+***Excluded namespaces configuration is only available in HNC v0.8 and later***
```

Comment:
I think italics alone would have been fine but I won't hold this up :) Feel free to remove the hold if you like.

Reply:
I'm using the same format as https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/docs/user-guide/concepts.md#exceptions-and-propagation-control in the same page. So I will remove the hold :)